d2renderers/d2svg/table.go

```diff
@@ -12,13 +13,42 @@ import (
 	"oss.terrastruct.com/util-go/go2"
 )
 
+func tableHeaderBorderRadius(shape d2target.Shape) string {
```

Comment:
can you leave a comment that this is also for `class` shapes, despite the name

Reply:
fixed, renamed this func, and add a comment above